index.js

```diff
@@ -99,30 +99,35 @@ UglifyWriter.prototype.processFile = function(inFile, outFile, relativePath, out
   var src = fs.readFileSync(inFile, 'utf-8');
   var mapName = path.basename(outFile).replace(/\.js$/,'') + '.map';
   var mapDir;
-  var origSourcesContent;
 
   if (this.sourceMapConfig.mapDir) {
     mapDir = path.join(outDir, this.sourceMapConfig.mapDir);
   } else {
     mapDir = path.dirname(path.join(outDir, relativePath));
   }
 
+  var sourceMapOptions = {
+    includeSources: this.options.sourceMapIncludeSources,
+    filename: inFile,
+    url: this.mapURL(mapName)
+  };
+
   var opts = {
-    fromString: true,
-    outSourceMap: this.mapURL(mapName),
-    enableSourcemaps: this.enableSourcemaps()
+    compress: this.options.compress,
+    mangle: this.options.mangle,
+    sourceMap: this.enableSourcemaps() ? sourceMapOptions : false
```

Comment:
Looking back at this, I'm not sure if this is what we want to do. It looks like I was trying to keep the input the options the same and map them to the new options that are in UglifyJS 3. Of course, now you can't access all of the properties anymore from your ember-cli-build config. Should I change this to bring all the options over again, with the expectation that anyone upgrading their ember-cli-uglify/broccoli-uglify-sourcemap would need to update their config?